Package@swift-5.9.swift

```diff
@@ -0,0 +1,45 @@
+// swift-tools-version: 5.9
+
+import PackageDescription
+
+let swiftSettings: [SwiftSetting] = [
+    .enableUpcomingFeature("ExistentialAny"),
```

Comment:
Won't we need this in 5.8 to make the change compile?

Reply:
No, it works down until 5.7.
But I think we might be able to just drop 5.8 all together, the generator is already at 5.9+.
I need to check.

Reply:
Yeah if the generator is 5.9 we can move to 5.9 only

Reply:
Sorry that was some misinformation, the generator is still on 5.8.
But again, it won't cause problems on 5.8.

Reply:
Alright i've updated the PR to be 5.9-only in anticipation of the 1.0.0 swift-openapi-generator release.